app/handlers/queue_worker.py

```
@@ -9,6 +9,7 @@
 from app.handlers.message_handler import MessageHandler
 from app.infrastructure.queues.supabase_queue import SupabaseQueue
 from app.core.config import settings
+from handlers.job_tracker import JobTracker, JobTrackerManager
```

Ruff (0.12.2)

`handlers.job_tracker.JobTracker` imported but unused

Remove unused import: `handlers.job_tracker.JobTracker`

(F401)